spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -113,7 +113,7 @@ definitions:
         type: u8
         desc: |
           Quality of the troposphere data. Encoded following RTCM DF389
-          specifcation but as TECU instead of m.
+          specifcation in units of m.
```

Comment:
is it meters?

Reply:
From the CLAS spec:
![image](https://user-images.githubusercontent.com/1215089/62508585-8bdbb300-b84b-11e9-9f94-dc39a76c3d48.png)

From RTCM:
![image](https://user-images.githubusercontent.com/1215089/62508686-de1cd400-b84b-11e9-94f4-6b9aab8e500a.png)

So they are basically reusing the SSR URA formula which takes the index (in the u8) and returns a value in metres.

We could try copying that formula into the YAML but it wouldn't look pretty.

I'm open to alternative wordings if you think there is something clearer we could use.